data/autoyast_sle15/autoyast_media_up_live_gnome.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE profile>
+<profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <add-on>
+    <add_on_products config:type="list">
+      <listentry>
+        <media_url><![CDATA[http://dist.suse.de/install/SLP/SLE-15-SP1-Module-Basesystem-TEST/x86_64/DVD1/]]></media_url>
+        <product>sle-module-basesystem</product>
+        <product_dir>/</product_dir>
+      </listentry>
+      <listentry>
+        <media_url><![CDATA[http://dist.suse.de/install/SLP/SLE-15-SP1-Module-Desktop-Applications-TEST/x86_64/DVD1/]]></media_url>
+        <product>sle-module-desktop-applications</product>
+        <product_dir>/</product_dir>
+      </listentry>
+      <listentry>
+        <media_url><![CDATA[http://dist.suse.de/install/SLP/SLE-15-SP1-Module-Server-Applications-TEST/x86_64/DVD1/]]></media_url>
+        <product>sle-module-server-applications</product>
+        <product_dir>/</product_dir>
+      </listentry>
+      <listentry>
+        <media_url><![CDATA[http://dist.suse.de/install/SLP/SLE-15-SP1-Module-Legacy-TEST/x86_64/DVD1/]]></media_url>
+        <product>sle-module-legacy</product>
+        <product_dir>/</product_dir>
+      </listentry>
+      <listentry>
+        <media_url><![CDATA[http://dist.suse.de/install/SLP/SLE-15-SP1-Module-Live-Patching-TEST/x86_64/DVD1/]]></media_url>
+        <product>sle-module-live-patching</product>
+        <product_dir>/</product_dir>
+      </listentry>
+    </add_on_products>
+  </add-on>
+  <bootloader>
+    <global>
+      <activate>false</activate>
+      <boot_extended>false</boot_extended>
+      <boot_mbr>true</boot_mbr>
+      <boot_root>true</boot_root>
+      <generic_mbr>false</generic_mbr>
+      <timeout config:type="integer">5</timeout>
+    </global>
+    <loader_type>grub2</loader_type>
+  </bootloader>
+  <general>
+    <mode>
+      <confirm config:type="boolean">false</confirm>
+      <second_stage config:type="boolean">false</second_stage>
+    </mode>
+    <signature-handling>
+      <accept_file_without_checksum config:type="boolean">true</accept_file_without_checksum>
```

Comment:
maybe we want "false" here ?

Reply:
Any document about this? I have checked several profile files in data/autoyast_sle15, this value is set as 'true'.

Reply:
I think values here are set up without any certain documentation just keeping in mind that it is **testing** profile. For example we blindly accepting GPG keys because it is fine for testing environment on other hand we want to fail if find unsigned file in repo because it will show us some bug in repo. 
My idea was that all files in testing repo should have checksum and we should test this. But if you saying that all other profiles having `true` here just ignore my comment

Reply:
Yes, I understand your opinion. While existing profiles with true here, so I won't change it. Thanks.